recipes/clark/1.2.2-b/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+#strictly use anaconda build environment
+CC=${PREFIX}/bin/gcc
```

Comment:
It should not be needed. It is my fault that they are there but I haven't had time to remove the lines.